scripts/exglobal_forecast.sh

```diff
@@ -67,6 +67,8 @@ NWPROD=${NWPROD:-${NWROOT:-$pwd}}
 HOMEgfs=${HOMEgfs:-$NWPROD}
 FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
 FIX_AM=${FIX_AM:-$FIX_DIR/fix_am}
+export FIX_AER=${FIX_AER:-$FIX_DIR/fix_aer}
+export FIX_LUT=${FIX_LUT:-$FIX_DIR/fix_lut}
```

Comment:
Based on these two settings I'm guessing you'll want me to put the new fix files into subfolders of $FIX_DIR called "fix_aer" and "fix_lut"? If so, these lines will resolve the removal of the other fix dir path settings in config.base. Let me know, thanks!